packages/content-conformance/src/rules.ts

```diff
@@ -150,8 +154,9 @@ export async function loadRule(
     // local rule - relative not found
   }
 
-  // ensure the rule severity level gets included so we can reference it when reporting
+  // ensure the rule severity level and any rule config get included so we can reference it when reporting
   rule.level = level
+  rule.config = ruleConfig
```

Comment:
This isn't currently validated at all. task to explore providing a config schema: https://app.asana.com/0/1100423001970639/1203414786494827/f